app/services/gitlab_integration/client_wrapper.rb

```diff
@@ -9,19 +9,31 @@ class ClientWrapper
       maintainer: 40
     }.freeze
 
-    def initialize(user_access_token)
-      @access_token = user_access_token
+    def initialize(user_access_token = nil)
+      @access_token = user_access_token || ENV["GITLAB_DEPLOYQA_BOT_TOKEN"]
     end
 
     def add_deployqa_bot_to_repo(repository, permission: :guest)
       permission_code = GITLAB_MEMBER_PERMISSIONS.fetch(permission)
+      deployqa_member = gitlab_client.team_members(repository.integration_id).find { |member| member.id == ENV["GITLAB_DEPLOYQA_BOT_ID"].to_i }
+
+      return if deployqa_member.present?
+
       gitlab_client.add_team_member(repository.integration_id, ENV["GITLAB_DEPLOYQA_BOT_ID"], permission_code)
     end
 
     def load_projects
       gitlab_client.projects(membership: true)
     end
 
+    def add_webhook_to_repo(repository)
+      gitlab_client.add_project_hook(repository.integration_id, "https://59a9f87e.ngrok.io/webhooks", merge_requests_events: 1)
```

Comment:
Please move to ENV

Reply:
And move `1` to const

Reply:
Since ENV variable is not enough to hande webhook route, in case of router.rb. Decided to just use rails url builder when gitlab webhooks handler route will be created.

For now url string leave empty. 

`1` is not a constant, it's true. Changed it to `true`